libbeat/dashboards/import_dashboards.go

```diff
@@ -91,6 +95,8 @@ func DefineCommandLine() (*CommandLine, error) {
 	cl.flagSet.BoolVar(&cl.opt.OnlyDashboards, "only-dashboards", false, "Import only dashboards together with visualizations and searches. By default import both, dashboards and the index-pattern.")
 	cl.flagSet.BoolVar(&cl.opt.OnlyIndex, "only-index", false, "Import only the index-pattern. By default imports both, dashboards and the index pattern.")
 	cl.flagSet.BoolVar(&cl.opt.Snapshot, "snapshot", false, "Import dashboards from snapshot builds.")
+	cl.flagSet.StringVar(&cl.opt.Certificate, "ca", "", "Certificate for SSL client authentication.")
```

Comment:
There should be three options like `cacert`, `cert`, and `key`. I chose those names because they are the same as `curl`. And curl also allows a key passphrase to be specified with the `pass` CLI flag.